UserManagementWithIdentity/Controllers/Api/UsersController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+using UserManagementWithIdentity.Models;
+
+namespace UserManagementWithIdentity.Controllers.Api
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [Authorize(Roles = "Admin")]
+    public class UsersController : ControllerBase
```

Comment:
First of all thank for your contribution :)
Please add the .vs folder to the .ignore file
regarding this controller please move the delete action to the normal Controller, as in .Net Core no need for api controller to handle this point